NEWS

```diff
@@ -1,17 +1,30 @@
 # Change log
 All notable changes to this project will be documented in this file.
 
+## [0.3.2] -- Unreleased
```

Comment:
If this is for 0.4.0, is it the case that this is now released, or is 0.32 still the unreleased one? If this is in fact 0.4.0, should we do a changelog entry for that?

Reply:
Also, should there be an entry for the "lifespan" concept, or will that be added to the changelog after doing the next release?

Reply:
I guess 0.3.2 was a dev version that was never released (I'm just calling it 0.4.0 for the actual release). see actual releases here: 

https://github.com/databio/simpleCache/releases

there is an entry for lifespan in there now. I had actually changed these things but hadn't savedthat commit, nice catch.